rita/src/main.rs

```diff
@@ -122,28 +127,20 @@ fn main() {
         });
     });
 
-    let mut dk = DebtKeeper::new(Int256::from(5), Int256::from(-10));
-
-    for debt_adjustment in rx {
-        match dk.apply_debt(debt_adjustment.ident, debt_adjustment.amount) {
+    for msg in debt_keeper_output {
```

Comment:
Why does this loop need to constantly roll, wouldn't it be better to block until something actually happens? That way this thread could just go to sleep on the socket within the debt keeper

Reply:
Pretty sure this does block, it is iterating over the channel output

Reply:
Oh, okay, my poor knowledge of Rust strikes again